ringpop.go

```diff
@@ -226,12 +258,23 @@ type BootstrapOptions struct {
 
 // Bootstrap starts the Ringpop
 func (rp *Ringpop) Bootstrap(opts *BootstrapOptions) ([]string, error) {
+
```

Comment:
Disagree. I think having a newline at the top is perfectly valid, if there is more than one block of code separated by newlines.

E.g. I would personally do:
```
func (f *Foo) Age() string {
    age := calcAge(f.dob) // No whitespace
    return age
}
```
```
func (f *Foo) SomethingComplicated() error {

    err := f.somethingComplicated
    if err != nil {
        return err
    }

    // But we should put whitespace at the top of this one because there are
    // two code blocks and it's more readable.
    return errors.New("Do you agree, Jeff?")
}
```